src/marqo/tensor_search/models/index_settings.py

```diff
@@ -68,7 +70,7 @@ def validate_url_pointer_treatment(cls, values):
             import warnings
             warnings.warn("'treatUrlsAndPointersAsImages' is deprecated. Use 'treatUrlsAndPointersAsMedia' instead.", DeprecationWarning)
```

Comment:
Where does this warning message go? Will user be able to see it?

Reply:
Actually right now the user can't see this, nor is it in marqo logs